docs/shared/link-directive.mdx

```diff
@@ -0,0 +1,13 @@
+Subgraph schemas "opt in" to Federation v2 features by applying the `@link` directive to the `schema` type. The `import` list of this definition must include each federation-specific directive that the subgraph schema uses. In the example below, the schema uses the `@key` and `@shareable` directives:
```

Comment:
Not sure if `"opt in"` requires the quotes, but I'm a minimalist when it comes to characters like `"` and `(` so take that with a grain of sale.